src/azul/chalice.py

```diff
@@ -80,6 +82,20 @@ class ServiceUnavailableError(ChaliceViewError):
     STATUS_CODE = 503
 
 
+@attr.s(auto_attribs=True, frozen=True, kw_only=True)
+class MetricThreshold:
+    lambda_name: str
+    handler_name: Optional[str] = attr.ib(default=None)
+    metric: str
+    value: int
+
+    # For the full list of supported metrics in the `AWS/Lambda` namespace, see:
+    #
+    # https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/aws-services-cloudwatch-metrics.html
+    #
+    metrics: ClassVar[tuple[str, ...]] = ('Errors', 'Throttles')
```

Comment:
An Enum would be more appropriate for this. Then you can also declare the `metric` attribute using that Enum instead of `str`.